src/components/App.tsx

```diff
@@ -42,9 +42,9 @@ function App() {
         </Box>
         <Router>
           <Routes>
-            <Route path="/" element={<SearchView />} />
+            <Route path='/' element={<SearchView />} />
```

Comment:
Wouldn't it be a good idea to have a separate route for the Search View Beta Version, either in this PR or another?